multi-agent/adapters/outbound/mongo/client_config_repository.py

```
@@ -47,15 +61,36 @@ def find_by_server(self, user_id: str, server_identifier: str) -> Optional[Clien
         return self._to_model(doc) if doc else None
 
     def save(self, user_id: str, config: ClientConfig) -> None:
```

Ruff (0.16.0)

[warning] Unused method argument: `user_id`

(ARG002)